Zend/zend_globals.h

```diff
@@ -232,8 +232,9 @@ struct _zend_executor_globals {
 	void *reserved[ZEND_MAX_RESERVED_RESOURCES];
 };
 
-#define EG_FLAGS_INITIAL	0x00
-#define EG_FLAGS_IN_SHUTDOWN	0x01
+#define EG_FLAGS_INITIAL	           (0)
+#define EG_FLAGS_IN_SHUTDOWN	       (1<<0)
```

Comment:
nit: This uses a mix of tabs and spaces to align the values. It's okay to use either all spaces or all tabs here.